From 0119fde5d7d3063c4e268cf94859154057710cd4 Mon Sep 17 00:00:00 2001
From: Edward Cross <edward.k.cross@gmail.com>
Date: Tue, 31 Jul 2018 13:59:25 +0200
Subject: [PATCH 2/3] Trebuchet: fix derp

 * a77601a96c9dc0f7489e87a6ab0bb2499b8a9461 made
  it load predictive apps through the SearchLauncherCallback
  instead of our implementation

Change-Id: I939adb6e0c4009cd22df4f81eb0ce5ecb30134ad
---
 src/com/android/launcher3/Launcher.java | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/com/android/launcher3/Launcher.java b/src/com/android/launcher3/Launcher.java
index b5c5cdeb4..b2c8763f9 100644
--- a/src/com/android/launcher3/Launcher.java
+++ b/src/com/android/launcher3/Launcher.java
@@ -3098,11 +3098,7 @@ public class Launcher extends BaseActivity
     public void tryAndUpdatePredictedApps() {
         List<ComponentKeyMapper<AppInfo>> apps = null;
         if (mSharedPrefs.getBoolean("pref_predictive_apps", true)) {
-            if (mLauncherCallbacks == null) {
-                apps = mPredictiveAppsProvider.getPredictions();
-            } else {
-                apps = mLauncherCallbacks.getPredictedApps();
-            }
+            apps = mPredictiveAppsProvider.getPredictions();
         }
 
         if (apps != null) {
-- 
2.17.1

